hbase-rsgroup/src/main/java/org/apache/hadoop/hbase/rsgroup/RSGroupAdminServer.java

```diff
@@ -385,7 +386,7 @@ public RSGroupInfoManager getRSGroupInfoManager() throws IOException {
   }
 
   @Override
-  public void removeServers(Set<Address> servers) throws IOException {
+  public void removeServers(@Nullable Set<Address> servers) throws IOException {
```

Comment:
I don't mean annotation `Nullable` is not good, but HBase doesn't use it around the codebases. So here makes me feel not unified, what about comment the nullable at the method @ parma description.

Reply:
Fair point, it does look awkward when very few places in the codebase use that. Removed it.